### ISSUE TITLE
2.51.1 breaks deploying from s3

### ISSUE THREAD

**Issue by nevins-b** [serverless/serverless#9724]
#9721 breaks using s3 as artifact storage

```
package:
  artifact: s3://blah/${self:service}/lambda.${self:custom.gitBranch}.${self:custom.gitCommit}.zip
```

causes the following error

```
Cannot access package artifact at "s3://blah/something/lambda.main.fe0cbe0.zip": ENOENT: no such file or directory, access '/Users/user/go/src/github.com/blah/something/deployments/s3:/blah/something/lambda.main.fe0cbe0.zip'
```

**Comment by nevins-b** [comment]
@medikoo

**Comment by medikoo** [comment]
@nevins-b Great thanks for reporting, will be fixed with #9725
